### ISSUE TITLE
Add an indicator to the card to show that the flight is paused

### ISSUE THREAD

**Issue by alexanderson1993** [Thorium-Sim/thorium-nova#553]
Knowing whether the flight is paused or not could really come in handy when messing around with thrusters or other things and them not properly responding.
                
### Version: 1.0.0-alpha.9

**Comment by mdonahoo** [comment]
Kind of like this?   I can make it bigger and more pronounced, but the logic hooks into the flight status from the lobby.

<img width="1916" height="911" alt="Image" src="https://github.com/user-attachments/assets/22c36f20-2ad1-4c1a-ae49-276ab2e5898f" />

**Comment by alexanderson1993** [comment]
Yeah, something like that is good, though I might make it even more prominent - in the center of the screen, with a transparent black background so all of the controls are covered and cannot be used.

**Comment by mdonahoo** [comment]
Ok.  I like the center and transparent black. 

<img width="1902" height="907" alt="Image" src="https://github.com/user-attachments/assets/2f8e0085-41fe-4d5d-8d17-ec59b7d72d4a" />
